docs/developers/guides/trace_transactions.md

```diff
@@ -0,0 +1,309 @@
+<!--
+order: 5
+-->
+
+# Trace Transactions
+
+Learn how to trace transactions and blocks on Evmos. {synopsis}
+
+This guide will cover the following topics:
+
+- [basic EVM tracing](#basic-evm-tracing)
```

Comment:
Would add the tech used for each section, e.g. `basic EVM tracing with JS` or `JSON-RPC debug_trace* endpoints`